aten/src/ATen/CMakeLists.txt

```diff
@@ -105,6 +105,7 @@ file(GLOB native_quantized_cpp
             "native/quantized/*.cpp"
             "native/quantized/cpu/*.cpp")
 file(GLOB native_nested_cpp "native/nested/*.cpp")
+file(GLOB native_transformer_cpp "native/transformers/*.cpp")
```

Comment:
I guess I personally wouldn't have bothered making a new folder cuz build system shenanigans, but I guess if you got it to work :)

Reply:
it would be nice if our build system didn't care so much about directory structure. this was much harder to get working than it would have been with buck.